loader/trampoline.c

```diff
@@ -73,10 +73,14 @@ LOADER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vkGetInstanceProcAddr(VkI
             // Old behavior is to return a global entrypoint regardless of the value of the instance handle.
             // Use new behavior if: The instance is valid and the minor version of the instance is greater than 1.2, which
             // was when the new behavior was added. (eg, it is enforced in the next minor version of vulkan, which will be 1.3)
+            // Maintain this behavior for vulkansc version 1.1.
```

Comment:
True, but I was assuming that it would exist in the future and that the VKSC 1.1 behavior here should match VK 1.3 behavior, but maybe that was a bad assumption on my part and VKSC 1.1 behavior should be the same as VKSC 1.0 and VK 1.2?